### PR TITLE
Added logic to create blanks

### PR DESCRIPTION
**Pull request by Yuriyhp**

### CODE REVIEW

**Review by WanDER33 — APPROVED** [review]

**Review by MykhailoKlapchuk — APPROVED** [review]

File: EPlast/EPlast.WebApi/Controllers/BlanksController.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        }
+
+        [HttpDelete("RemoveBiographyDocument/{documentId}")]
+        [Authorize(AuthenticationSchemes = "Bearer")]
+        public async Task<IActionResult> RemoveDocument(int documentId)
+        {
+            await _blankBiographyDocumentService.DeleteFileAsync(documentId);
+            _logger.LogInformation($"Document with id {{{documentId}}} was deleted.");
+
+            return Ok();
```
Comment (voroniak) [inline EPlast/EPlast.WebApi/Controllers/BlanksController.cs:59]:
```suggestion
            return NoContent();
```

File: EPlast/EPlast.WebApi/Controllers/BlanksController.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        }
+
+        [HttpGet("BiographyDocumentBase64/{fileName}")]
+        [Authorize(AuthenticationSchemes = "Bearer")]
+        public async Task<IActionResult> GetFileBase64(string fileName)
+        {
+            var fileBase64 = await _blankBiographyDocumentService.DownloadFileAsync(fileName);
+
+            return Ok(fileBase64);
```
Comment (voroniak) [inline EPlast/EPlast.WebApi/Controllers/BlanksController.cs:68]:
```suggestion
            return Ok(await _blankBiographyDocumentService.DownloadFileAsync(fileName));
```

File: EPlast/EPlast.WebApi/Controllers/BlanksController.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            return Ok(document);
+        }
+
+        [HttpGet("GetDocumentByUserId/{userId}")]
+        [Authorize(AuthenticationSchemes = "Bearer")]
+        public async Task<IActionResult> GetDocumentByUserId(string userId)
+        {
+           var document =  await _blankBiographyDocumentService.GetDocumentByUserId(userId);
+
+            return Ok(document);
```
Comment (voroniak) [inline EPlast/EPlast.WebApi/Controllers/BlanksController.cs:49]:
```suggestion
            return Ok( await _blankBiographyDocumentService.GetDocumentByUserId(userId));
```

File: EPlast/EPlast.WebApi/Controllers/BlanksController.cs [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        /// <param name="document">An information about a specific document</param>
+        /// <returns>A newly created Blank document</returns>
+        [HttpPost("AddDocument/{userId}")]
+        [Authorize(AuthenticationSchemes = "Bearer")]
+        public async Task<IActionResult> AddDocument(BlankBiographyDocumentsDTO document)
+        {
+            await _blankBiographyDocumentService.AddDocumentAsync(document);
+            _logger.LogInformation($"Document with id {{{document.ID}}} was added.");
+
+            return Ok(document);
```
Comment (voroniak) [inline EPlast/EPlast.WebApi/Controllers/BlanksController.cs:40]: return Created(.....)

File: EPlast/EPlast.BLL/Services/Blank/BlankBiographyDocumentsService.cs [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+            _repositoryWrapper.BiographyDocumentsRepository.Delete(document);
+            await _repositoryWrapper.SaveAsync();
+        }
+
+        public async Task<string> DownloadFileAsync(string fileName)
+        {
+            var fileBase64 = await _blankFilesBlobStorage.GetBlobBase64Async(fileName);
+
+            return fileBase64;
```
Comment (voroniak) [inline EPlast/EPlast.BLL/Services/Blank/BlankBiographyDocumentsService.cs:60]:
```suggestion
            return await _blankFilesBlobStorage.GetBlobBase64Async(fileName);
```

**Review by ostappppppp — APPROVED** [review]

**Pull request merged**